src/steps/custom-object-field-equals.ts

```diff
@@ -88,7 +95,7 @@ export class CustomObjectFieldEqualsStep extends BaseStep implements StepInterfa
       }
 
       // Field validation
-      if (this.compare(operator, queryResult.result[0][field], expectedValue)) {
+      if (this.compare(operator, queryResult.result[0][field].toString(), expectedValue)) {
```

Comment:
- [ ] Is this `.toString()` intended, or an artifact from debugging?

Reply:
I had to `toString()` for boolean values. I also had in mind what if numbers were compared and remembered numbers and dates are already handled in util via `isNumber` or `regex.test`